webui/src/Components/ConfigurationForm.tsx

```diff
@@ -141,9 +142,13 @@ const AttrsOfControl = (props: AttrsOfControlProps) => {
                 <FormControl mr={4} mb={4}>
                   <FormLabel>{key}</FormLabel>
                   {typeof value == 'boolean' ? (
-                    <CustomCheckbox name={jp.stringify([...keys, item, key])} checked={value} />
+                    <CustomCheckbox name={jp.stringify([...keys, item, key])} isChecked={value || false} />
```

Comment:
`isChecked={value || false}`

Do I want to know why the `|| false` is necessary?

Reply:
I lost faith in life while fighting the typechecker

Reply:
I premused.

Carry on.